src/main/java/org/breedinginsight/brapi/v2/BrAPITrialsController.java

```diff
@@ -38,19 +39,21 @@
 public class BrAPITrialsController {
 
     private final String referenceSource;
+    private final SecurityService securityService;
     private final BrAPITrialService experimentService;
     private final ExperimentQueryMapper experimentQueryMapper;
 
     @Inject
-    public BrAPITrialsController(BrAPITrialService experimentService, ExperimentQueryMapper experimentQueryMapper, @Property(name = "brapi.server.reference-source") String referenceSource) {
+    public BrAPITrialsController(SecurityService securityService, BrAPITrialService experimentService, ExperimentQueryMapper experimentQueryMapper, @Property(name = "brapi.server.reference-source") String referenceSource) {
+        this.securityService = securityService;
         this.experimentService = experimentService;
         this.experimentQueryMapper = experimentQueryMapper;
         this.referenceSource = referenceSource;
     }
 
     @Get("/trials{?queryParams*}")
     @Produces(MediaType.APPLICATION_JSON)
-    @ProgramSecured(roleGroups = {ProgramSecuredRoleGroup.ALL})
+    @ProgramSecured(roleGroups = {ProgramSecuredRoleGroup.PROGRAM_SCOPED_ROLES})
     public HttpResponse<Response<DataResponse<List<BrAPITrial>>>> getExperiments(
```

Comment:
Needs to allow Experimental Collaborator role access. You could do that in the PR or else I'll do it in BI-2258 where I've implemented filtering in this method.